python/paddle/base/data_feed_desc.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, proto_file):
 
     def set_batch_size(self, batch_size):
         r"""
-        Set :attr:`batch_size` in :ref:`api_base_DataFeedDesc` . :attr:`batch_size` can be changed during training.
+        Set :attr:`batch_size` in :ref:`api_paddle_DataFeedDesc` . :attr:`batch_size` can be changed during training.
```

Comment:
`DataFeedDesc` is not a public API. So do not use reference
```suggestion
        Set :attr:`batch_size` in ``paddle.base.DataFeedDesc`` . :attr:`batch_size` can be changed during training.
```

Reply:
OK